ldap/servers/slapd/back-ldbm/ldif2ldbm.c

```diff
@@ -360,7 +360,14 @@ int
 ldbm_back_upgradednformat(Slapi_PBlock *pb)
 {
     struct ldbminfo *li = NULL;
+    int task_flags;
     slapi_pblock_get(pb, SLAPI_PLUGIN_PRIVATE, &li);
+    slapi_pblock_get(pb, SLAPI_TASK_FLAGS, &task_flags);
+
+    if (task_flags & SLAPI_TASK_RUNNING_FROM_COMMANDLINE) {
+        dblayer_setup(li);
```

Comment:
I wonder if it is worth putting a check for the return code here so that if it fails, it won't fail silently.
Something like:

```
        if (dblayer_setup(li)) {
            slapi_log_err(SLAPI_LOG_CRIT, "ldbm_back_upgradednformat", "dblayer_setup failed\n");
            return -1;
        }
```

Reply:
absolutely right. Thanks